core/templates/pages/contributor-dashboard-page/services/translate-text.service.ts

```diff
@@ -121,22 +129,22 @@ export class TranslateTextService {
   }
 
   private _getUpdatedTextToTranslate(
-      text: string | string[],
+      text: string | string[] | null,
       more: boolean,
       status: Status,
-      translation: string
+      translation: string | string[]
   ): TranslatableItem {
     const {
       dataFormat,
       contentType,
       interactionId,
       ruleType
     }: {
-      dataFormat?: string,
-      contentType?: string,
-      interactionId?: string,
-      ruleType?: string
-    } = this.stateAndContent[this.activeIndex] || {};
+      dataFormat: string,
+      contentType: string,
+      interactionId: string | null,
+      ruleType: string | null
```

Comment:
Why can these be null?

Reply:
![image](https://user-images.githubusercontent.com/86483059/134001835-59851c0b-31d8-4085-be10-21100fe99089.png)
So I added null as a type to them.

Reply:
Yes, but you should explain why they can be `null`, not in a way that "because TS check fail" but instead "they can be null because when they have no interaction these will be null" (not sure if that is the actual reason).

Reply:
Updated.

Reply:
Done